style/golf.mss

```diff
@@ -0,0 +1,76 @@
+#landcover[zoom >= 12] {
+  ::high-zoom[zoom >= 12] {
+    [feature = 'golf_tee'],
+    [feature = 'golf_fairway'],
+    [feature = 'golf_rough'],
+    [feature = 'golf_driving_range'] {
+      polygon-fill: @grass;
+    }
+  }
+  ::high-zoom[zoom >= 13] {
+    [feature = 'golf_green'] {
+      polygon-fill: @pitch;
+    }
+    [feature = 'golf_bunker'] {
+      polygon-fill: @sand;
+    }
+  }
+  ::high-zoom[zoom >= 15] {
+    [feature = 'golf_rough'] {
+      polygon-pattern-file: url('symbols/golf_rough.svg');
+      polygon-pattern-comp-op: soft-light;
+    }
+  }
+}
+
+#golf-line[zoom >= 16] {
+  [golf = 'hole'] {
+    line-color: @address-color;
+    line-width: 0.5;
+  }
+}
+
+#text-line[zoom >= 16] {
+  [feature = 'golf_hole'][ref != ''],
+  [feature = 'golf_hole'][name != ''] {
+    text-placement: line;
+    text-size: 11;
+    text-fill: @address-color;
+    text-face-name: @book-fonts;
+    text-halo-radius: @standard-halo-radius;
+    text-halo-fill: @standard-halo-fill;
+    text-name: "";
+
+    [ref != ''] { text-name: "[ref]"; }
+    [name != ''] { text-name: "[name]"; }
```

Comment:
Set one of these as default, then check if that one is null, and if so, set the other.

Reply:
Updated.